src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/ValidationRulePart.cshtml

```diff
@@ -24,35 +64,23 @@
             window.validatorObject.push(obj);
         }
         $(elementId).bind('change keyup', function (e) {
-            var validateResult = false;
-            if (window.validator.isEmpty(option)) {
-                validateResult = window.validator[type]($(elementId).val());
-            }
-            else if (window.validator.isJSON(option)) {
-                validateResult = window.validator[type]($(elementId).val(), JSON.parse(option));
-            }
-            else {
-                validateResult = window.validator[type]($(elementId).val(), option);
-            }
-            var validationElement = document.getElementById(validationElementId);
-            if (validateResult) {
-                if (validationElement) {
-                    $(elementId).closest('form').find(':submit').removeAttr('disabled');
-                    validationElement.remove();
-                }
-            }
-            else {
-                if (!validationElement) {
-                    var validationMessage =  '@T["{0} validation failed", type]';
-                    var dangerText = '<div id="' + validationElementId + '" class="text-danger">' + validationMessage + '</div>';
-                    $(elementId).after(dangerText);
-                    $(elementId).closest('form').find(':submit').attr('disabled', 'true');
-                }
-            }
+            validateEvent(type, validationElementId, option, elementId, validationMessage, defaultValidateMessage);
         })
+        var submitBtn = $(elementId).closest('form').find(':submit')[0];
+        if (submitBtn) {
+            if (isIE) {
+                submitBtn.attachEvent('click', function () {
+                    validateEvent(type, validationElementId, option, elementId, validationMessage, defaultValidateMessage);
+                })
+            } else {
+                submitBtn.addEventListener('click', function (e) {
+                    validateEvent(type, validationElementId, option, elementId, validationMessage, defaultValidateMessage);
+                })
+            }
+        }
     }
     if ('@type' != 'none') {
-        validateElementFunction('@type', '@elementId', '@option');
+        validateElementFunction('@type', '@elementId', '@option','@validationMessage');
```

Comment:
If you fix one of the validation errors then you'll be able to submit the form without further client-side validation errors. Server-side validation will work but I think you should still get client-side validation there.

https://user-images.githubusercontent.com/1976647/114313815-6743de00-9af8-11eb-8678-89229bf0c844.mp4